jwst/pipeline/calwebb_coron3.cfg

```diff
@@ -4,6 +4,7 @@ class = "jwst.pipeline.Coron3Pipeline"
     [steps]
       [[stack_refs]]
       [[align_refs]]
-      [[klip]]
+        config_file = align_refs.cfg
```

Comment:
Similar to the previous comment of removing defaults, the line with `config_file` should be removed. Having the steps listed is fine, but otherwise no other default specifications.